RadarSDK/RadarAPIHelper.m

```diff
@@ -76,6 +76,9 @@ - (void)requestWithMethod:(NSString *)method
             void (^dataTaskCompletionHandler)(NSData *data, NSURLResponse *response, NSError *error) = ^(NSData *data, NSURLResponse *response, NSError *error) {
                 if (error) {
                     dispatch_async(dispatch_get_main_queue(), ^{
+                        [[RadarLogger sharedInstance]
+                            logWithLevel:RadarLogLevelError
+                                message:[NSString stringWithFormat:@"Received network error | error = %@", error]];
```

Comment:
I wonder whether I should audit what thread(s) `logWithLevel()` is getting called on. 🤔